### PR TITLE
WIP Test merge fork

### PR DESCRIPTION
**Pull request by cyrilchampier**

### CODE REVIEW

**Comment by cyrilchampier** [comment]
fail: i wanted to do the opposite, sorry :)

File: lib/segments/pid.rb [hunk 1]
```diff
@@ -11,13 +11,7 @@ class HL7::Message::Segment::PID < HL7::Message::Segment
   add_field :patient_dob do |value|
     convert_to_ts(value)
   end
-  add_field :admin_sex do |sex|
```
Comment (cyrilchampier) [inline lib/segments/pid.rb]: @adaedra pkoi ?

**Comment by mitch-lindsay** [comment]
Some of those additional segments would be useful if you want to put them up as a PR individually.

**Comment by cyrilchampier** [comment]
Hi, yes, I will first merge your diffs into our repository, and then I will open a second PR to merge back our modifications (in few weeks) :)

**Pull request closed without merging**